code/src/pages/dashboard/dashboard-orders/DashboardOrdersPage.tsx

```diff
@@ -48,6 +57,13 @@ const DashboardOrdersPage = () => {
       <AppTypography translationKey="dashboardTabs.orders.filters.title" />
     );
 
+  const paginationBlock =
+    (totalPages ?? 0) > 1 ? (
+      <AppContainer className="dashboard-orders-tab__toolbar-pagination-container">
+        <AppPagination page={page} count={totalPages} size="large" />
+      </AppContainer>
+    ) : null;
```

Comment:
I guess here is better to use `&&` operator for render like below suggestion
In this approach nothing happened if the value of condition is `falsy` it's `null`

My suggestion show `paginationBlock` only when it's `true`

```suggestion
const paginationBlock = totalPages > 1 && (
  <AppContainer className="dashboard-orders-tab__toolbar-pagination-container">
    <AppPagination page={page} count={totalPages} size="large" />
  </AppContainer>
)
```

Reply:
Thank you! fixed